### PR TITLE
 Fixed ipns address resolution in fuse unix mount

### PR DESCRIPTION
**Pull request by kvm2116**

### CODE REVIEW

**Comment by Stebalien** [comment]
This looks correct. Could you add the signoff trailers?

**Comment by kvm2116** [comment]
Added the License and Sign-Off trailers.

File: fuse/ipns/ipns_unix.go [hunk 1]
```diff
@@ -196,7 +196,8 @@ func (s *Root) Lookup(ctx context.Context, name string) (fs.Node, error) {
 	}
 
 	// other links go through ipns resolution and are symlinked into the ipfs mountpoint
-	resolved, err := s.Ipfs.Namesys.Resolve(s.Ipfs.Context(), name)
+	ipns_name := "/ipns/" + name
```
Comment (Kubuxu) [inline fuse/ipns/ipns_unix.go]: I would prefer if there was no underscore in the name.

**Comment by Stebalien** [comment]
Dammit git! Sorry, for some reason git never seems to understand that when I push from a branch, I mean to push *that* branch (not some other branch because reasons).

New PR: https://github.com/ipfs/go-ipfs/pull/5327

**Pull request closed without merging**